### ISSUE TITLE
Horus x12s - Fw: 1.6.04 - Battery voltage values tx shown incorrect wt 3 cells 18650...

### ISSUE THREAD

**Issue by Lombardoa** [FrSky-OS/HORUS-X10-X10S-X12S-bug-report#241]
On my Horus x12s FROS, I replaced the NI-MH battery pack with three 18650 li-ion cells. The radio works normally, while the value read on transmitter battery voltage its wrong ... I have seen that the values above 10.7V are wrong. While below this value the values read are correct.
Please, can you verify?
Thanks In advance ...

**Comment by Lombardoa** [comment]
Hello Adela...
why not reproduced...you can remove the battery , then check with a power supply at 12V...easy!!!

**Comment by Adelaxie** [comment]
yes, we have good display here.. how is the value wrong?pic?

**Comment by Lombardoa** [comment]
Hi Adela,
well, I'm sending you 6 photos where you can see the voltage problem. In 
the right corner, below, you can see the time.
Photo n.1, starting condition, 11.7V
Photo n.2, after about 25min. 11.3V
Photo n.3, after about 55min, 11.2V
*Photo n.4, after about 1 hour, 11.8V ... here the voltage rises, and I 
don't know why ... this is the real problem.*
Photo No. 5, the voltage continues to decrease normally, then stop 
everything and go to sleep.
In Photo No. 6, you will find all the info about the firmware.

I advise you to remove the battery, power the tx with a variable power 
supply. Set it to 13V, then turn on the radio, calibrate the value read 
on the horus, and then start decreasing the voltage from 13V to 6V.
Try it several times ... if you find inaccuracies, please fix everything.
I thank you for your patience and cooperation.

Arturo

Il 20/05/2019 04:29, Adela ha scritto:

**Comment by Lombardoa** [comment]
![Pic-1](https://user-images.githubusercontent.com/50540767/58131273-8a242b00-7c1e-11e9-8079-9eefa97f7911.jpg)
![Pic-2](https://user-images.githubusercontent.com/50540767/58131310-a2944580-7c1e-11e9-92c2-70c08e344b5b.jpg)
![Pic-3](https://user-images.githubusercontent.com/50540767/58131323-a88a2680-7c1e-11e9-97e0-048eef1e88f7.jpg)
![Pic-4](https://user-images.githubusercontent.com/50540767/58131333-ade77100-7c1e-11e9-95a8-8973d932ad6c.jpg)
![Pic-5](https://user-images.githubusercontent.com/50540767/58131345-b6d84280-7c1e-11e9-9588-1c2bfb359f7f.jpg)
![Pic-6-InFO](https://user-images.githubusercontent.com/50540767/58131351-ba6bc980-7c1e-11e9-9031-c87957a7888d.jpg)

**Comment by Adelaxie** [comment]
testing

**Comment by Adelaxie** [comment]
@Lombardoa Hi,sorry, we have been testing on this for days bu we still cannot reproduce what you reported, can you check with the real voltage  by removing the battery out and measuring by voltagemeter?

**Comment by Lombardoa** [comment]
Hi Adela,
as you suggested, I measured the voltage simultaneously with two multimeters. One on the output pins of the external module, Ph4.
And the other, on the battery output connector. To increase the battery discharge, I enabled internal RF and also mp3 music. Below are the results.
1) Start with Ph1 (12V - time 19:13)
2) after a while Ph2 (11.6V - time 20:00)
3) after a while Ph3 (12.2V - Time 20:07) (here the value rises)
At this point the problem comes from the balancing circuit of the new LI-Ion batteries. 
The case is solved.
Thank you again for your cooperation and patience ... see you soon ... Arturo

![Ph1](https://user-images.githubusercontent.com/50540767/58614981-3074ce00-82ba-11e9-9820-d94c2ccbee88.jpg)
![Ph2](https://user-images.githubusercontent.com/50540767/58614994-38347280-82ba-11e9-9e3d-6e8030bc8409.jpg)
![Ph3](https://user-images.githubusercontent.com/50540767/58615008-42ef0780-82ba-11e9-9bb2-ae4d73599d46.jpg)
![Ph4](https://user-images.githubusercontent.com/50540767/58615017-4bdfd900-82ba-11e9-8131-f66c5c25f439.jpg)

**Comment by Lombardoa** [comment]
Hi Adela,
as you suggested, I measured the voltage simultaneously with two multimeters. One on the output pins of the external module, Ph4.
And the other, on the battery output connector. To increase the battery discharge, I enabled internal RF and also mp3 music. Below are the results.
1) Start with Ph1 (12V - time 19:13)
2) after a while Ph2 (11.6V - time 20:00)
3) after a while Ph3 (12.2V - Time 20:07) (here the value rises)
At this point the problem comes from the balancing circuit of the new LI-Ion batteries. 
The case is solved.
Thank you again for your cooperation and patience ... see you soon ... Arturo

![Ph1](https://user-images.githubusercontent.com/50540767/58614981-3074ce00-82ba-11e9-9820-d94c2ccbee88.jpg)
![Ph2](https://user-images.githubusercontent.com/50540767/58614994-38347280-82ba-11e9-9e3d-6e8030bc8409.jpg)
![Ph3](https://user-images.githubusercontent.com/50540767/58615008-42ef0780-82ba-11e9-9bb2-ae4d73599d46.jpg)
![Uploading Ph4.jpg…]()
